scripts/dragen_reprocessing/get_dragen_metrics.py

```diff
@@ -0,0 +1,149 @@
+"""Get all needed metrics from a dragen run.
+
+Usage:
+    > python3 get_dragen_metrics.py -o DRAGEN_OUTPUT -s SAMPLE_NAME STAGING_AREA_PATH"""
```

Comment:
can we call it DRAGEN_OUTPUT_PATH just so we know its a path

Reply:
👍